src/app/app.js

```diff
@@ -0,0 +1,44 @@
+
+/**
+ * Dependencies.
+ */
+
+var React = require('react');
+var Fluxible = require('fluxible');
+var routrPlugin = require('fluxible-plugin-routr');
+var pageRegistry = require('./page-registry');
+
+/**
+ * Create fluxible app.
+ */
+
+var app = new Fluxible({
+  appComponent: React.createFactory(require('./app-component'))
```

Comment:
I guess I simply disagree. It seems like it would make it less clear what's going on while bloating the file with unnecessary variables.